go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/jackc/pgx/v4 v4.18.1
 	github.com/labstack/echo/v4 v4.10.2
 	github.com/labstack/gommon v0.4.0
+	github.com/pelletier/go-toml v1.9.5
```

Comment:
I wonder if we should use github.com/BurntSushi/toml instead. We use this one in composer. The API should be very similar.